From e14d724a932a255eec4d60b08e128519c33e88ee Mon Sep 17 00:00:00 2001
From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Mon, 25 Feb 2019 11:59:23 +0100
Subject: [PATCH] meson: declare version.h as dependency for systemd

This is a followup to #11815 and adds the last missing dependency.
With this #11565 is hopefully really fixed.

Upstream-Status: Backport

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 232f3e1..70148be 100644
--- a/meson.build
+++ b/meson.build
@@ -1618,7 +1618,8 @@ executable('systemd',
            include_directories : includes,
            link_with : [libcore,
                         libshared],
-           dependencies : [threads,
+           dependencies : [versiondep,
+                           threads,
                            librt,
                            libseccomp,
                            libselinux,
-- 
2.7.4

